jaggery-modules/carbon-metrics/feature/pom.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ ~ Copyright (c) 2009-2011, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Thanks @thushara35 , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/599d0b1561ccafb683b7f43d394952442ffce229)